### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by numel007**
Removed get_one() and some unused dict appends

### CODE REVIEW

**Comment by numel007** [comment]
Now ready to merge into dev

File: src/scraper.py [hunk 1]
```diff
@@ -87,53 +84,38 @@ def get_points_change(self):
...
-        self.dict['volume'] = self.volume
         return self.volume
 
     
     def get_avg_volume(self):
         '''Parse self.parse_avg_volume for average volume, return self.avg_volume'''
 
         self.avg_volume = (self.parse_avg_volume.find('span')).string
-        self.dict['avg_volume'] = self.avg_volume
+        # self.dict['avg_volume'] = self.avg_volume
```
Comment (pondorasti) [inline src/scraper.py]: Any reason why this line has been commented out and not deleted?

Comment (numel007) [reply src/scraper.py]: Now fixed in [this commit](https://github.com/BoringStocks/StonkScraper/pull/9/commits/1f82422ee1c49c1cd4a364028882ab38111bb49a)

**Review by pondorasti — APPROVED** [review]
Cleaning code once in a while is great practice!

**Pull request merged**